keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -591,6 +588,18 @@ service Provisioner {
 This call is meant to retrieve the unique provisioner Identity. This identity will have to be set in `BucketRequest.Provisioner` field in order to invoke this specific provisioner.
 
 ```
+message Protocol {
+	// ProtocolName is the name of the protocol
```

Comment:
Is this indentation supposed to be like this for the comment?

Reply:
It appears to be used consistently in the kep. I don't know if this is the convention elsewhere.